source/features/branch-buttons.tsx

```diff
@@ -61,7 +61,7 @@ async function getTagLink() {
 	return link;
 }
 
-async function getDefaultBranchLink() {
+async function getDefaultBranchLink(): Promise<HTMLElement | SVGElement | undefined> {
```

Comment:
```suggestion
async function getDefaultBranchLink(): Promise<HTMLAnchorElement | undefined> {
```

Reply:
This can't be HTMLAnchorElement. This kind of comes back to some of the JSX issues I mentioned way back, https://github.com/sindresorhus/refined-github/pull/1783#issuecomment-464412087.

`JSX.Element` is just an alias for `HTMLElement | SVGElement`, so I'll use that for now.

Reply:
We can't use `HTMLElement | undefined` either?

Reply:
Doesn't seem to work. Like I said before, the JSX issue is still kind of around. We have a somewhat working solution with types, but it's so so. I gotta 😴 but I'll look at this some more tomorrow.